ui/cypress/e2e/explorer.test.ts

```diff
@@ -349,8 +349,7 @@ describe('DataExplorer', () => {
     })
   })
 
-  // todo: investigate flakiness of this test: https://github.com/influxdata/influxdb/issues/16330
-  describe.skip('raw script editing', () => {
+  describe('raw script editing', () => {
```

Comment:
i'd prefer this to stay skipped, i have another pr that is handling this: https://github.com/influxdata/influxdb/pull/16392.

Reply:
@hoorayimhelping I did run them a few times but I have no problem skipping them here in favor of your PR

Reply:
@Asalem1 I think they're okay to not be skipped now, I have what I think is the fix in master. I was more curious about the root cause of this issue - I'm having a hard time figuring out why removing `buildActiveQuery` had an effect on these tests and I was wondering if you found anything out in your digging

Reply:
@hoorayimhelping aw, I see. Truthfully, I'm not sure I fully comprehend why the test was failing when that line was added, and why it's no longer failing. I imagine that there are unintended consequences to the functionality of the utility function that may have been effecting the query builder / script editor that we are currently unaware of.